src/main/java/org/aksw/iguana/cc/worker/ResponseBodyProcessor.java

```diff
@@ -72,7 +72,7 @@ public List<LanguageProcessor.LanguageProcessingData> getResponseDataMetrics() {
             return responseDataMetrics;
         }
 
-        LOGGER.info(MessageFormat.format("Shutting down ResponseBodyProcessor with {0} min timeout to finish processing. {1} tasks remaining.", timeout.toMinutes() + "." + (timeout.toSecondsPart() / (double) 60), executor.getQueue().size()));
+        LOGGER.info(MessageFormat.format("Shutting down ResponseBodyProcessor with {0} min timeout to finish processing. {1} tasks remaining.", timeout.toMinutes() + "." + (timeout.toSecondsPart() / 6), executor.getQueue().size()));
```

Comment:
Why divide by 6? This seems more like it should be:

```suggestion
        LOGGER.info(MessageFormat.format("Shutting down ResponseBodyProcessor with {0}min {1}s timeout to finish processing. {2} tasks remaining.",
                timeout.toMinutes(), 
                String.format("%02d", timeout.toSecondsPart() % 60),
                executor.getQueue().size()));
                ```